docs/verify/__use_estimator.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Estimator primitive
+description: How to compute an expectation value with the `Estimator` primitive
+---
+
+# Compute an expectation value with `Estimator` primitive
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) primitive.
```

Comment:
I fixed all relative links like this. I'm not going to call them all out though. I checked that the URLs work in live preview.